### PR TITLE
Upgrading jetty http client in consumers module

### PR DESCRIPTION
**Pull request by druminski**

### CODE REVIEW

File: hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/PublishingServlet.java [hunk 1]
```diff
@@ -138,6 +142,7 @@ private void handlePublishAsynchronously(HttpServletRequest request, HttpServlet
                                         new BrokerListenersPublishingCallback(listeners)));
 
                     } catch (InvalidMessageException | ConvertingException | UnsupportedContentTypeException exception) {
+                        logger.warn("Error while parsing message for topic {}", topic.getQualifiedName(), exception);
```
Comment (adamdubiel) [inline hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/PublishingServlet.java]: wont it kill us with amount of logs?

**Comment by adamdubiel** [comment]
:+1:

**Comment by pawelszymczyk** [comment]
:+1:

File: hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/PublishingServlet.java (hunk 1 above)
Comment (druminski) [inline hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/PublishingServlet.java]: Ok, fixed to log only root cause. Full stack will be logged in dry run mode

**Pull request merged**